model/workflow.go

```diff
@@ -206,34 +206,6 @@ func (w *Workflow) setDefaults() {
 	}
 }
 
-// WorkflowRef holds a reference for a workflow definition
-type WorkflowRef struct {
```

Comment:
Isn't better keep the Workflow here?

Reply:
I think it's fine as it is. It's the same logic as moving `functionRef` to `Action`. This is the only place where `workflowRef` and `functionRef` is used. It's bound to actions.

Reply:
This type is only used by Action struct, so i organize them together (in the same file)